recipes/nginx.rb

```diff
@@ -91,7 +90,7 @@
       mode '0644'
       variables(
         http_port: port,
-        server_name: site_opts['server_name']
+        server_name: site_opts['server_alias'].empty? ? site_name : site_opts['server_alias'].first
```

Comment:
because server name was arbitrary, we can't have it be laeroiyhlaihglehg for instance.  The aliases list should contain a list of FQDNs for the deployed site, with a fallback on site_name if none are set.  We need this to support vhosts properly.